app/client/src/components/pages/Community/components/tabs/Protect.js

```diff
@@ -1202,10 +1223,13 @@ function Protect() {
 
                           {allProtectionProjects.length > 0 && (
                             <>
-                              <p>
+                              <div css={modifiedInfoBoxStyles}>
+                                <strong>
+                                  {allProtectionProjects.length.toLocaleString()}
+                                </strong>{' '}
                                 EPA funded protection projects in the{' '}
                                 <em>{watershed}</em> watershed.
-                              </p>
+                              </div>
```

Comment:
The wrapping `<div css={modifiedInfoBoxStyles}>` is correct, but there should be a paragraph inside as well... I just realized we should to update two other pages to have the correct structure as well:
- the Community Overview component
- the WaterbodyList component

If you do a search for `<div css={modifiedErrorBoxStyles}>` you'll see those all use the correct nested structure of div > p.

Reply:
Good catch! I went ahead and resolved both of these issues.

Reply:
thanks!